service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -35,18 +32,7 @@
     private static final String MIGRATION_ID_KEY = "migrationId";
     private final GeneratedOrderRemovalAction generatedOrderRemovalAction;
 
-    private final DraftOrderService draftOrderService;
-
-    private static final List<Long> casesWithUploadDraftCMOs = List.of(
-        1609255930041507L, 1606987766244887L, 1604311638012517L, 1610617556504448L, 1611053637858331L,
-        1611831571219051L, 1612440806991994L, 1611613172339094L, 1597234670803750L, 1610638275960711L,
-        1612792241936277L, 1608227716601946L, 1606816433160806L, 1604488701821402L, 1612260529074989L,
-        1602684664094829L, 1611162463937927L, 1603112430374845L, 1609332375073780L, 1606920447453254L,
-        1603186601101963L, 1601047439875628L, 1607700990753436L, 1593751852182152L, 1599567482297796L,
-        1600763666484840L, 1608550125506632L, 1611073519965954L, 1601036838115652L, 1600681449523496L,
-        1604665756707715L, 1605864027624648L, 1604698798063326L, 1601466075213425L, 1606835150877657L,
-        1600924712703100L, 1602695126592366L, 1610448987787679L, 1604057238263139L, 1607650998329954L,
-        1602072210447867L, 1607089490115787L, 1605252849637867L);
+    private final SealedCMORemovalAction sealedCMORemovalAction;
```

Comment:
The screenshot in the Jira ticket shows that the CMO is in `draft` state

Reply:
`SealedCMORemovalAction` has a method - `removeDraftCaseManagementOrder`. Bare in mind that it was previously named `cmoRemovalAction` so this needs cleaned up at some point. Not part of this migration PR however.